activesupport/lib/active_support/notifications/instrumenter.rb

```diff
@@ -57,7 +57,8 @@ class Event
 
       def self.clock_gettime_supported? # :nodoc:
         defined?(Process::CLOCK_PROCESS_CPUTIME_ID) &&
-          !Gem.win_platform?
+          !Gem.win_platform? &&
+          !RUBY_PLATFORM.include?("solaris")
```

Comment:
Is this need to consider the upper case? I'm not sure if it is really necessary but seems Ruby consider it.
https://github.com/ruby/ruby/blob/77596fb7a91cc119b25ac9e19b3c8682709765b4/test/drb/drbtest.rb#L14 
https://github.com/ruby/ruby/blob/77596fb7a91cc119b25ac9e19b3c8682709765b4/test/ruby/test_process.rb#L1691 
https://github.com/ruby/ruby/blob/77596fb7a91cc119b25ac9e19b3c8682709765b4/test/readline/test_readline.rb#L620

Reply:
Thanks, that's great advice. I couldn't find any other code doing this. I've updated it as you suggested.